index.html

```diff
@@ -131,6 +131,10 @@ <h2>The Club</h2>
           <td><a href="https://github.com/burntcarrot/quackspeak">quackspeak</a></td>
           <td>a text-to-speech engine using ducks, for fun</td>
         </tr>
+        <tr>
+          <td><a href="https://github.com/Trusted97/whitecat">Whitecat</a></td>
+          <td>a developer toolbox for avoid boring setup during PHP development, for fun</td>
```

Comment:
can you change "developer toolbox for avoid boring setup during PHP development" to be 5 words or less? (I need to make that clearer in the PR template because people keep missing that.)

Reply:
WDYT now?  @seven1m

Reply:
Well it's 6 words lol, but it's fine for now. I can tweak it later.